content/docs/rfcs/57/README.md

```diff
@@ -0,0 +1,325 @@
+This mapping is similar in nature to the shard cluster allocation in [52/WAKU2-RELAY-STATIC-SHARD-ALLOC](/spec/52/).
+Shard indices allocated in that way are in the range `16 - 127`.
+The Status CC community uses index `16` (not to confuse with shard cluster index `16`, which is the Status shard cluster).
+
+### Owner Mapping
+
+> *Note*: This way of mapping will be specified post-MVP.
+
+Community owners can choose to map their communities to any shard within the index range `128 - 767`.
+
+## 1:1 Chat
+
+[55/STATUS-1TO1-CHAT](/spec/55) uses partitioned topics to map 1:1 chats to a set of 5000 content topics.
+This document extends this mapping to 8192 content topics that are, in turn, mapped to 128 shards in the index range of `768 - 895`.
+
+```
+contentPartitionsNum = 8192
+contentPartition = mod(publicKey, contentPartitionsNum)
+partitionContentTopic = "contact-discovery-" + contentPartition
+
+partitionContentTopic = keccak256(partitionContentTopic)
+
+shardNum = 128
+shardIndex = 768 + mod(publicKey, shardNum)
+
+```
+
+# Infrastructure Nodes
+
+As described in [30/ADAPTIVE-NODES](/spec/30/),
+Waku supports a continuum of node types with respect to available resources.
+Infrastructure nodes are powerful nodes that have a high bandwidth connection and a high up-time.
+
+This document, which informs about simple ways of scaling Status over Waku,
+assumes the presence of a set of such infrastructure nodes in each shard.
+Infrastructure nodes are especially important for providing connectivity in the roll-out phase.
+
+Infrastructure nodes are not limited to Status fleets, or nodes run by community owners.
+Anybody can run infrastructure nodes.
+
+## Statically-Mapped Communities
+
+Infrastructure nodes are provided by the community owner, or by members of the respective community.
+
+## Owner-Mapped Communities
+
+Infrastructure nodes are part of a subset of the shards in the range `128 - 767`.
+Recommendations on choosing this subset will be added in a future version of this document.
+
+Status fleet nodes make up a part of these infrastructure nodes.
+
+## 1:1 chat
+
+Infrastructure nodes are part of a subset of the shards in the range `768 - 985` (similar to owner-mapped communities).
+Recommendations on choosing this subset will be added in a future version of this document.
+
+Desktop clients can choose to only use filter and lightpush.
+
+> *Note*: Discussion: I'd suggest to set this as the default for the MVP.
+The load on infrastructure nodes would not be higher, because they have to receive and relay each message anyways.
+This comes as a trade-off to anonymity and decentralization,
+but can significantly improve scaling.
+We still have k-anonymity because several chat pairs are mapped into one content topic.
+We could improve on this in the future, and research the applicability of PIR (private information retrieval) techniques in the future.
+
+
+# Infrastructure Shards
+
+## Control Message Shards
```

Comment:
If I understand correctly, general control messages will still be relayed _by default_ in the main Community shard? This section just discusses "special" control message sharding, for large messages. For clarity it may be worth being explicit here (or earlier) about which traffic goes to the static community shard ("all" by default), and that this separate sharding is not a requirement for all control messages? E.g. renaming this section to `Sharding for large Control Messages`. Hoping this makes sense. 😅

Reply:
Makes sense! In other words, the way I read this RFC is that it starts with general description of static shard allocation per community and moves to specific further shard separation techniques which may or may not be required based on community size/need.

Reply:
Yes! Looks good and is clear. Thanks.